src/sentry/tasks/post_process.py

```diff
@@ -203,6 +205,22 @@ def post_process_group(
                 ):
                     safe_execute(callback, event, futures)
 
+            commit_cache_key = "workflow-owners-ingestion:org-{}-has-commits".format(
+                event.project.organization_id
+            )
+            commit_data = cache.get(commit_cache_key)
+            if commit_data is None:
+                org_has_commits = Commit.objects.filter(
+                    organization_id=event.project.organization_id
+                ).exists()
+                commit_data = {"has_commits": org_has_commits}
+                cache.set(commit_cache_key, commit_data, 3600)
```

Comment:
We could just store `org_has_commits` in here without the dict, since `has-commits` is already built into the name. Unless you had plans to store other data here later?